python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -79,6 +79,12 @@ def is_builtin_len(func):
     return False
 
 
+def is_builtin_zip(func):
+    if is_builtin(func) and func.__name__ == 'zip':
+        return True
+    return False
```

Comment:
`return is_builtin(func) and func.__name__ == 'zip'`

Reply:
Done